e2e/playwright/pay.spec.ts

```diff
@@ -0,0 +1,359 @@
+import operation from "./assets/operationLDC.json";
+
+test.setTimeout(90_000);
+
+const URL = "http://localhost:3000";
+
+const TEST_EMAIL = "test@test.com";
+const DEFAULT_WAIT_TIME = 300;
+
+test.describe("Payment flow", () => {
+  const USER_EMAIL = "test@test.com";
+
+  test.beforeAll(async () => {
+    await insertTestUser(USER_EMAIL);
+
+    await gqlAdmin(
+      `
+        mutation InsertTeams(
+          $teams: [teams_insert_input!]!, 
+        ) {
+          insert_teams(
+            objects: $teams,
+          ) {
+            affected_rows
+          }
+        }
+      `,
+      {
+        teams: [team],
+      }
+    );
+
+    await gqlAdmin(
+      `
+        mutation InsertFlows(
+          $flows: [flows_insert_input!]!,
+        ) {
+          insert_flows(
+            objects: $flows,
+          ) {
+            affected_rows
+          }
+        }
+      `,
+      {
+        flows: [ldcflow],
+      }
+    );
+
+    await gqlAdmin(
+      `
+        mutation InsertPublishedFlow(
+          $publishedFlows: [published_flows_insert_input!]!
+        ) {
+          insert_published_flows(
+            objects: $publishedFlows
+          ) {
+            affected_rows
+          }
+        }
+    `,
+      {
+        publishedFlows: publishedFlows,
+      }
+    );
+
+    await gqlAdmin(
+      `
+        mutation InsertOperations($operations: [operations_insert_input!]!) {
+          insert_operations(objects: $operations) {
+            affected_rows
+          }
+        }
+      `,
+      {
+        operations: [operation],
+      }
+    );
+  });
+
+  test.afterAll(async () => {
+    await gqlAdmin(`
+      mutation CleanDatabase {
+        delete_session_backups(where: {}){
+          affected_rows
+        }
+        delete_analytics_logs(where: {}){
+          affected_rows
+        }
+        delete_analytics(where: {}){
+          affected_rows
+        }
+        delete_published_flows(where: {}){
+          affected_rows
+        }
+        delete_operations(where: {}){
+          affected_rows
+        }
+        delete_flows(where: {}) {
+          affected_rows
+        }
+        delete_teams(where: {}) {
+          affected_rows
+        }
+      }
+    `);
+  });
+
+  test.skip("Should pay and send the application", async ({ page }) => {
+    await navigateToPayComponent({ page, mode: "unpublished" });
+
+    await page.getByText("Pay using GOV.UK Pay").click();
+
+    await page.waitForResponse(
+      (resp) => resp.url().includes("create-send-events") && resp.ok()
+    );
+
+    expect(await page.getByText("Application sent").textContent()).toBeTruthy();
+  });
+
+  test("Should pay within GOV.UK and send the application", async ({
+    page,
+  }) => {
+    await navigateToPayComponent({ page, mode: "preview" });
+
+    await page.getByText("Pay using GOV.UK Pay").click();
+
+    // GOV.UK Pay
+    await page
+      .getByRole("textbox", { name: "Card number" })
+      .fill("4444333322221111");
```

Comment:
Could we store the mock card numbers as variables and link to [the docs](https://docs.payments.service.gov.uk/testing_govuk_pay/#mock-card-numbers-and-email-addresses) please? 

Can we also have one number for each "Testing outcome" as defined by Gov UK (eg "success", "failure because card declined" etc)?